### PR TITLE
Update hello-world.md

### PR DESCRIPTION
**Pull request by seneca**
Minor changes

### CODE REVIEW

File: packages/nuejs.org/docs/hello-world.md [hunk 1]
```diff
@@ -140,7 +141,7 @@ We start by creating a [custom layout file](custom-layouts.html) called `layout.
 
 
 ### Add shared data
-Then we add a data file called `site.yaml` with all the site-wide data to fill in our personal information and other basic data being used on our header and footer:
+We then add a data file named site.yaml, containing all the site-wide data, to populate our personal information and other essential details used in the header and footer:
```
Comment (nobkd) [inline packages/nuejs.org/docs/hello-world.md]:
```suggestion
We then add a data file named `site.yaml`, containing all the site-wide data, to populate our personal information and other essential details used in the header and footer:
```

**Review by nobkd — CHANGES_REQUESTED** [review]

File: packages/nuejs.org/docs/hello-world.md (hunk 1 above)
Comment (seneca) [reply packages/nuejs.org/docs/hello-world.md]: Thanks for spotting the missing back ticks between site.yaml.

File: packages/nuejs.org/docs/hello-world.md [hunk 2]
```diff
@@ -74,17 +75,17 @@ Let's add a new file to our project folder called "blog.css", which will take ca
 touch blog.css
 ```
 
-Nue automatically adds the following line on the HTML without the need to reload your page:
+Nue automatically adds the following line in the HTML without the need to reload your page:
```
Comment (nobkd) [inline packages/nuejs.org/docs/hello-world.md]: Maybe `to` fits better? What do you think?

Comment (nobkd) [reply packages/nuejs.org/docs/hello-world.md]:
```suggestion
Nue automatically adds the following line to the HTML without the need to reload your page:
```

**Review by nobkd — APPROVED** [review]

**Review by tipiirai — APPROVED** [review]

**Pull request merged**